docs/developer_docs/release_workflow.md

```diff
@@ -0,0 +1,33 @@
+# libpymcr release workflow
+
+The release workflow is mostly automated using continuous integration builds,
+but some actions and triggering a release needs to be done manually by the developer.
+
+To create a release.
+
+1. Create a branch and edit the `CHANGELOG.md` and `CITATION.cff` files to update it with a new tag.
```

Comment:
From the extra info below it looks like it might be clearer to say update the version number as opposed to tag (or is there more to update?)

Reply:
The version number is the minimum required but I guess you should also update the blurb in the `CHANGELOG.md` file - although I guess this should be updated when each individual issues / PR are merged.